api/TrustedHTML.json

```diff
@@ -46,8 +48,60 @@
           "deprecated": false
         }
       },
+      "toJSON": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TrustedHTML/toJSON",
```

Comment:
Wow, https://developer.mozilla.org/docs/Web/API/TrustedHTML/toJSON actually exists :)

Commented on something at https://github.com/mdn/content/pull/2978#pullrequestreview-686087283...

Reply:
We have a few toJSON and toString (stringifier) pages. Like for the iterable pages, they all have a close structure that make them fairly quick to create.

Reply:
IMHO, for them to be useful, they would have to say which attributes are actually serialized in the resulting JSON, since that's often defined by prose in specs and isn't really guessable from looking at https://developer.mozilla.org/en-US/docs/Web/API/TrustedHTML, for example.